### PR TITLE
Add new repo with 8 tests

### PR DESCRIPTION
**Pull request by anirudh711**
The following tests are added from the repository - https://github.com/json-iterator/java
[com.jsoniter.extra.TestPreciseFloat#test_indirect_encode](https://github.com/json-iterator/java/blob/6925cf4c19d313504b416f58a349a36bf563e0e1/src/test/java/com/jsoniter/extra/TestPreciseFloat.java#L23)
[com.jsoniter.output.TestGson#test_serializeNulls](https://github.com/json-iterator/java/blob/6925cf4c19d313504b416f58a349a36bf563e0e1/src/test/java/com/jsoniter/output/TestGson.java#L89)
[com.jsoniter.output.TestObject#test_omit_default](https://github.com/json-iterator/java/blob/6925cf4c19d313504b416f58a349a36bf563e0e1/src/test/java/com/jsoniter/output/TestObject.java#L354)
[com.jsoniter.output.TestObject#test_omit_null](https://github.com/json-iterator/java/blob/6925cf4c19d313504b416f58a349a36bf563e0e1/src/test/java/com/jsoniter/output/TestObject.java#L228)
[com.jsoniter.output.TestNested#test_map_of_objects](https://github.com/json-iterator/java/blob/6925cf4c19d313504b416f58a349a36bf563e0e1/src/test/java/com/jsoniter/output/TestNested.java#L85)
[com.jsoniter.output.TestNested#test_object_of_array](https://github.com/json-iterator/java/blob/6925cf4c19d313504b416f58a349a36bf563e0e1/src/test/java/com/jsoniter/output/TestNested.java#L59)
[com.jsoniter.output.TestGenerics#test_wildcard](https://github.com/json-iterator/java/blob/6925cf4c19d313504b416f58a349a36bf563e0e1/src/test/java/com/jsoniter/output/TestGenerics.java#L47)
[com.jsoniter.output.TestObject#test_indention](https://github.com/json-iterator/java/blob/6925cf4c19d313504b416f58a349a36bf563e0e1/src/test/java/com/jsoniter/output/TestObject.java#L282)

Path to nondex logs in vm - /home/ar80/nondex-logs/json-iterator-java/nondex_one_mode.log

### CODE REVIEW

**Comment by darko-marinov** [comment]
@zzjas can check these lines. Because it's opened less than 4 hours before the deadline, we may not get to accept on time for this progress report. But it can count for the next one.

**Comment by anirudh711** [comment]
Sure, thankyou!

File: format_checker/forked-projects.json [hunk 1]
```diff
@@ -443,5 +443,7 @@
   "https://github.com/zalando/riptide": "unforked",
   "https://github.com/zeroturnaround/gradle-jrebel-plugin": "unforked",
   "https://github.com/zhangxd1989/spring-boot-cloud": "unforked",
-  "https://github.com/zhurlik/gradle-jboss-modules-plugin": "unforked"
+  "https://github.com/zhurlik/gradle-jboss-modules-plugin": "unforked",
+  "https://github.com/json-iterator/java": "unforked"
+
```
Comment (zzjas) [inline format_checker/forked-projects.json:448]: Can you remove the extra empty line? Thanks.

Comment (anirudh711) [reply format_checker/forked-projects.json]: Deleted the line and commited

**Comment by darko-marinov** [comment]
`forked-project.json` has to be sorted. Please change that and also ideally update the checker.

**Comment by anirudh711** [comment]
I have updated the checker, I will raise a separate PR since that needs a review. Please review and merge this PR. Thanks!

**Comment by darko-marinov** [comment]
Accepting this per Campuswire, but a separate PR that would have sorted the projects and added a checker would have been better.

**Pull request merged**